.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release Instant Seal Node With Deployed Schemas Docker Image
+on:
+  push:
+    tags: '*'
```

Comment:
We should make it trigger off of tags that lead off with docker/ just to give the blockchain folks a clean space to do their own versioning for tags